packages/ra-ui-materialui/package.json

```diff
@@ -62,6 +62,7 @@
         "clsx": "^1.1.1",
         "css-mediaquery": "^0.1.2",
         "dompurify": "^2.4.3",
+        "hotscript": "^1.0.12",
```

Comment:
Because **you** asked me to put it in dependencies ^^
![image](https://github.com/marmelab/react-admin/assets/1122076/2b01f3da-3e4d-4a2c-a1af-dbe61dd34be8)

Reply:
bump